script/backport-patchset.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+
+import os
+import re
+import sys
+import urllib2
+
+SOURCE_ROOT = os.path.abspath(os.path.dirname(os.path.dirname(__file__)))
+PATCHES_DIR = os.path.join(SOURCE_ROOT, 'patches')
+
+def main():
```

Comment:
The script overall LGTM and very handy.

It does, though, assume a lot of prior knowledge into how we do chromium backports. It would be nice for future contributors if `len(sys.argv) == 1` returned a brief help explaining how, when, and why this should be used.